lib/nav-header/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component, PropTypes } from 'react';
+
+require('./style.css');
+
+export default class NavHeader extends Component {
+  render () {
+    const { articleContent, backToSearch } = this.props;
+    return (
+      <div className='navBarContainer'>
+          <a className='backButton' onClick={() => backToSearch()}><i className="fa fa-angle-left fa-2x" aria-hidden="true"></i></a>
```

Comment:
same here. It would be more readable to spread this out over multiple lines with each prop on a separate line